docs/UserGuide.md

```diff
@@ -629,34 +629,27 @@ Format: `list`
 
 ### Finding patients: `find`
 
-Finds patients whose names contain any of the given keywords.
-
-Format: `find [KEYWORD]… [n/NAME]… [p/PHONE]… [e/EMAIL]… [a/ADDRESS]… [t/TAG]… [c/CONDITION]… [d/TASK_DESCRIPTION]… [m/MEDICATION]… [r/REMARK]…`
+Finds patients whose parameters contain the given keywords.
 
+Format: `find [KEYWORD]… [xyz/XYZ_KEYWORD]…`
+* `xyz/XYZ_KEYWORD` refers to a keyword for a specific parameter.
 * There should be at least one parameter for the command.
 * The search is case-insensitive. e.g `hans` will match `Hans`.
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.
 * Partial words can be matched e.g. `Han` will match `Hans`.
-* Patients matching at least one keyword in every parameter type will be returned (i.e. AND search for different parameters, OR search for same parameter). In more details,
+* Patients matching at least one keyword in every parameter will be returned (i.e. AND search for different parameters, OR search for same parameter). In more details,
     * At least one of the patient's details (name, phone, email, address, tag, condition, task description, or medication) must match with at least one `KEYWORD`.
```

Comment:
Left out remark.
Not a mistake on your part, but I'm enforcing a certain order of the attributes for consistency, though I have forgotten to change some of them.
```suggestion
    * At least one of the patient's details (name, phone, email, address, tag, task description, condition, medication, or remark) must match with at least one `KEYWORD`.
```